CONTRIBUTING.md

```diff
@@ -0,0 +1,45 @@
+# Contributing
+
+When contributing to this service,
+please open an issue on
+[GitHub issues](https://github.com/onOfficeGmbH/sdk/issues)
+or send a mail to `apisupport@onoffice.de`
+this service before making a change.
+
+## Bugs
+
+Bugs must be reported via the
+[GitHub issues](https://github.com/onOfficeGmbH/sdk/issues)
+Bug fixes and improvements can be provided via a Pull Request.
+
+A fix that redefines that behaviour of an implementation are considered
+as a [breaking change](#breaking-changes).
+
+## Features
+
+Features and Breaking Changes must be discussed with the
+maintainers of this repository.
+
+Features that also changes the behaviour of existing implementations
```

Comment:
"existing implementations" in terms of "current usages" or "existing setups"? I would rethink the terminology here (and above).

Features that also change~~s~~ the behaviour [...]

Reply:
IMO `current usages` and `existing setups` wouldn't be enough to clearify what is meant here.

"current usage" would indicate to much the users perspective, but implemenations can also be the internal behaviour. Like internal classes interact with each other or how class works internally.
The same goes for "existing setups". The description is to close to the enduser, while ignoring the internal environment.

But maybe that's just me 😄 

@dkaisers what do you think?